config/config_test.go

```diff
@@ -620,3 +712,12 @@ func newTempFileWithContent(content string) (*os.File, error) {
 	tmpf.Close()
 	return tmpf, nil
 }
+
+func containsString(slice []string, contains string) bool {
```

Comment:
[nit] The variable `contains` reminds me a function returning a boolean, I do prefer `expected`  or something like that.